main/influxdb_watcher.go

```diff
@@ -184,8 +207,14 @@ func main() {
 			break
 		}
 	}
+	for {
+		databaseExists := influxdbBroker.checkDatabaseExists(time.Duration(15) * time.Second)
```

Comment:
do we need to set this 15 seconds as an env variable?

Reply:
I totally could, but it still is just going to check and recheck until it sees the database exists.